js/stores/__tests__/message_store_test.js

```diff
@@ -0,0 +1,37 @@
+jest.dontMock('../message_store');
+jest.dontMock('object-assign');
+jest.dontMock('react/lib/keyMirror');
+
+describe('MessageStore', function() {
+
+  var AppDispatcher;
+  var MessageStore;
+  var callback;
+  
+  var ActionTypes = require('../../constants/constants').ActionTypes;
+
+  var actionCreateMessage = {
+    type: ActionTypes.NEW_MESSAGE,
+    data: {
+      user: "Bob",
+      text: "Hello, everyone I'm new.",
+      timestamp: new Date().getTime()
+    }
+  };
+
+  beforeEach(function() {
+    AppDispatcher = require('../../dispatcher/app_dispatcher');
+    MessageStore = require('../message_store');
+    callback = AppDispatcher.register.mock.calls[0][0];
```

Comment:
See the Testing Stores section http://facebook.github.io/react/blog/2014/09/24/testing-flux-applications.html It's a trick for getting the callback the MessageStore registered with the dispatcher.